skfusion/tests/test_dfmc.py

```diff
@@ -84,6 +84,19 @@ def postprocessor(data):
         np.testing.assert_almost_equal(fuser.complete(relation), trnf)
         np.testing.assert_equal(fusion_graph.get_relation('R').data, R12)
 
+    def test_reproducibility(self):
```

Comment:
Without this fix `test_reproducibility` occasionally fails depending on the [iteration order](https://github.com/mims-harvard/scikit-fusion/blob/master/skfusion/fusion/decomposition/_init.py#L13) of `obj_types` in matrix initialization. With the proposed fix this test always finishes successfully.